docs/source/quickstart/model_config.md

```diff
@@ -27,4 +35,13 @@ layers:
     name: final_dense
     units: 1
     activation: null
-```
\ No newline at end of file
+optimizer: 
+  key: adam
+  gradient_clip_value: 5.0
```

Comment:
@mohazahran Make sure you have default values for these if not explicitly overriden by the users. Same for `lr_schedule`

Reply:
Does it make any difference having SGD/Adam/... with cyclical learning rate schedulers? Or the lr value of each batch it overwritten (I would imagine this is the case)?

Reply:
Yeah, I'm not sure how Adam works with cyclic learning rate really. I should read about that. My understanding was cyclic learning rate is typically used with SGD.

@mohazahran do you know if using cyclic learning rate actually works? I think this is the kind of stuff we should set up unit tests for. If tensorflow's implementation of optimizers changes, we can easily catch it.

Reply:
I don't think lr schedule require a specific optimizer for it to work. So it shouldn't matter if it's SGD/Adam/..
I didn't test for it to actually see the lr decaying though.

Reply:
I agree with @mohazahran. The scheduler overrides the values of the optimizer.